data/campaigns/Sceptre_of_Fire/scenarios/2p5_Reaching_the_Runecrafter.cfg

```diff
@@ -0,0 +1,357 @@
+#textdomain wesnoth-sof
+[scenario]
+    name= _ "Reaching the Runecrafter"
+    id=2p5_Reaching_the_Runecrafter
```

Comment:
Just wondering: what does 2p5 even mean? Is it supposed to be an attempt to avoid renumbering the other scenarios? If so, I would have thought a more intuitive prefix could be used instead. Also the scenario count for this campaign remains at 9 instead of 10...